trunk/SUAVE/Components/Energy/Converters/Combustor.py

```diff
@@ -254,6 +261,107 @@ def compute_rayleigh(self,conditions):
         self.outputs.fuel_to_air_ratio       = f    
         self.outputs.mach_number             = M_out
     
+    def compute_scramjet(self,conditions):
+        """ This computes the output values from the input values according to
+        equations from the source.
+
+        Assumptions:
+        JP-7 used as fuel, fixed output Cp and gamma
+
+
+        Source:
+        Heiser, William H., Pratt, D. T., Daley, D. H., and Unmeel, B. M.,
+        "Hypersonic Airbreathing Propulsion", 1994
+
+        Inputs:
+        conditions.freestream.
+          isentropic_expansion_factor          [-]
+          specific_heat_at_constant_pressure   [J/(kg K)]
+          temperature                          [K]
+          stagnation_temperature               [K]
+          universal_gas_constant               [J/(kg K)] 
+        self.inputs.
+          stagnation_temperature               [K]
+          stagnation_pressure                  [Pa]
+          inlet_nozzle                         [-]
+
+        Outputs:
+        self.outputs.
+          stagnation_temperature               [K]
+          stagnation_pressure                  [Pa]
+          stagnation_enthalpy                  [J/kg]
+          fuel_to_air_ratio                    [-]
+          static_temperature                   [K]
+          static_pressure                      [Pa]
+          velocity                             [m/s]
+          mach_number                          [-]         
+
+        Properties Used:
+          self.fuel_data.specific_energy       [J/kg]
+          self.efficiency                      [-]
+          self.axial_fuel_velocity_ratio       [-]
+          self.fuel_velocity_ratio             [-]
+          self.burner_drag_coefficient         [-]
+          self.temperature_reference           [-]
+          self.absolute_sensible_enthalpy      [J/kg]
+          self.specific_heat_constant_pressure [J/(kg K)]
+        """         
+        # unpack the values
+        
+        # unpacking the values from conditions
+        R      = conditions.freestream.universal_gas_constant
+        
+        # unpacking the values form inputs
+        nozzle = self.inputs.inlet_nozzle
+        Pt_in   = self.inputs.stagnation_pressure
+
+        
+        # unpacking values from self
+        htf    = self.fuel_data.specific_energy
+        eta_b  = self.efficiency
+        Vfx_V3 = self.axial_fuel_velocity_ratio
+        Vf_V3  = self.fuel_velocity_ratio
+        CfAwA3 = self.burner_drag_coefficient
+        Tref   = self.temperature_reference
+        hf     = self.absolute_sensible_enthalpy
+        Cpb    = self.specific_heat_constant_pressure
+        
+        # New flow properties
+        Cpb         = 1510.
```

Comment:
Do not hardcode these values. I am not sure what these are since they don't have descriptive names. Depending on what they mean we should set them somewhere else.